Sources/TimeFreeze.swift

```diff
@@ -20,10 +21,23 @@ struct TimeFreeze {
         return (TimeFreeze.systemUptime() - self.uptime) + self.timestamp
     }
 
+    /// Returns the time plus analytical metadata about the time.
+    ///
+    /// See explanation of fields in the comment for the FullTime struct fore more info.
+    var fullTime: FullTime? {
```

Comment:
I'd expose a `var timeSinceLastNTPSync` as opposed to this `FullTime` new struct

Reply:
can't do this bc it's racy, explanation below